assets/css/edd-admin.css

```diff
@@ -4060,3 +4093,4 @@ select.edd-form-group__input {
 	line-height: initial;
 	margin: 8px 0 0;
 }
+
```

Comment:
@LisaCee Very strangely this extra new line at the end of the file is stopping the file from being loaded in my environment. Not really sure why but when I remove it everything is fine. Otherwise I get:

<img width="948" alt="Screen Shot 2020-09-25 at 1 42 21 PM" src="https://user-images.githubusercontent.com/491124/94299000-09252400-ff35-11ea-8e6e-33c107d307ef.png">

Reply:
Could it be a difference in line ending format (Windows vs Linux/Unix?)

Reply:
I _thought_ most files were supposed to end in a new line, but I can remove it if it's causing issues.  I can't duplicate it here in Chrome or FF.

Reply:
Could be, but it works fine on my other machine which is using generally the same local environment. Lots of random suggestions here: https://stackoverflow.com/questions/58215104/whats-the-neterr-http2-protocol-error-about

I've seen it on/off on random things locally the past few weeks but have never been able to nail it down to anything until the empty line here.

Reply:
For some reason I have two blank lines at the end of my copy of the file (4096 and 4097) but it's not causing issues here.

Reply:
So, do we think it should stay or should it go?